src/code/z_scene.c

```diff
@@ -232,7 +232,7 @@ void Scene_CommandSpecialFiles(PlayState* play, SceneCmd* cmd) {
     }
 
     if (cmd->specialFiles.cUpElfMsgNum != 0) {
-        play->cUpElfMsgs = Play_LoadFile(play, &sNaviMsgFiles[cmd->specialFiles.cUpElfMsgNum - 1]);
+        play->elfMessageList = Play_LoadFile(play, &sNaviMsgFiles[cmd->specialFiles.cUpElfMsgNum - 1]);
```

Comment:
The names here are kinda chaos lol

`play->elfMessageList`
gets data from `gOverworldNaviMsgs`/`gDungeonNaviMsgs` (which btw why are those `g`-prefixed),
which are from files `elf_message_field`/`elf_message_ydan`, which are referenced in `sNaviMsgFiles`,
which is indexed by `cUpElfMsgNum - 1`

I know the PR doesn't want to scope creep out of just PlayState things but I feel like `elfMessageList` is going to get renamed anyway? (e.g. to "naviMsgs" or smth like that)

Reply:
I would like to see it renamed, but yea i didnt want to deal with that mess. it should probably just all be made consistent at one time as a separate pr